<ui version="4.0" >
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QCheckBox" name="red_led" >
      <property name="text" >
       <string>Red LED</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Config</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Crystal:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QDoubleSpinBox" name="crystal" >
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>27.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Base frequency:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QDoubleSpinBox" name="freq" >
         <property name="maximum" >
          <double>1000.000000000000000</double>
         </property>
         <property name="value" >
          <double>915.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Actual:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="freq_actual" >
         <property name="text" >
          <string>000.000000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Channel:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QSpinBox" name="channel" >
         <property name="maximum" >
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2" >
        <widget class="QPushButton" name="send" >
         <property name="text" >
          <string>&amp;Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Commands</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="idle" >
         <property name="text" >
          <string>&amp;Idle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="cal" >
         <property name="text" >
          <string>&amp;Calibrate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="tx" >
         <property name="text" >
          <string>&amp;TX</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QPushButton" name="rx" >
         <property name="text" >
          <string>&amp;RX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Sweep</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Start:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QDoubleSpinBox" name="sweep_start" >
         <property name="maximum" >
          <double>1000.000000000000000</double>
         </property>
         <property name="value" >
          <double>902.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Stop:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="sweep_stop" >
         <property name="maximum" >
          <double>1000.000000000000000</double>
         </property>
         <property name="value" >
          <double>928.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>Step:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QDoubleSpinBox" name="sweep_step" >
         <property name="maximum" >
          <double>1000.000000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="sweep" >
         <property name="text" >
          <string>&amp;Go</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
